component/org.wso2.carbon.identity.oauth2.dpop/src/main/java/org/wso2/carbon/identity/oauth2/dpop/constant/DPoPConstants.java

```diff
@@ -45,20 +44,12 @@ public class DPoPConstants {
     public static final String CNF = "cnf";
     public static final String TOKEN_TYPE = "token_type";
     public static final String JWK_THUMBPRINT = "jkt";
+    public static final String DPOP_JKT_TABLE_NAME = "IDN_OAUTH2_DPOP_JKT";
+    public static final String DPOP_JKT = "dpop_jkt";
     public static final String AUTHORIZATION_HEADER = "authorization";
+    public static final String CLIENT_ID = "client_id";
+    public static final String AUTHORIZATION_CODE_GRANT_TYPE = "authorization_code";
     public static final String OAUTH_REVOKE_ENDPOINT = "/oauth2/revoke";
     public static final String SKIP_DPOP_VALIDATION_IN_REVOKE = "skip_dpop_validation_in_revoke";
     public static final boolean DEFAULT_SKIP_DPOP_VALIDATION_IN_REVOKE_VALUE = true;
-
-    /**
-     * This class defines SQLQueries.
-     */
-    public static class SQLQueries {
```

Comment:
Why are we removing this?

Reply:
moved to a seperate class : `SQLQuaries`